src/components/footer.tsx

```diff
@@ -0,0 +1,147 @@
+      ],
+    },
+    {
+      header: "SPORENIE",
+      items: [
+        {
+          link: "/savings#savings-accounts",
+          text: "SPORIACE ÚČTY",
+        },
+        {
+          link: "/wages#net-salary",
+          text: "STAVEBNÉ SPORENIE",
+        },
+        {
+          link: "/savings#retirement-savings",
+          text: " SPORENIE NA DÔCHODOK",
+        },
+        {
+          link: "/investments",
+          text: "INVESTÍCIE ",
+        },
+      ],
+    },
+    {
+      header: "DANE",
+      items: [
+        {
+          link: "/taxes#income-tax",
+          text: "DAŇ Z PRÍJMU",
+        },
+        {
+          link: "/taxes#dph",
+          text: "DPH ",
+        },
+      ],
+    },
+  ];
+
+  return (
+    <footer className="bg-white dark:bg-gray-900">
+      <hr className="my-6 border-gray-200 sm:mx-auto dark:border-gray-700 lg:my-3 w-full max-w-screen-xl" />
+      <div className="mx-auto w-full max-w-screen-xl p-4 py-6 lg:py-8">
+        <div className="md:flex md:justify-between">
+          <div className="flex gap-5 px-6 flex-col pb-10 md:pb-0 md:text-start text-center">
+            <div>
+              <p className="text-gray-500 dark:text-gray-400 font-medium">
+                Objav viac jednoduchých a intuitívnych nástrojov na našej
+                stránke
+              </p>
+            </div>
+            <div className="flex md:justify-start justify-center items-center gap-28 md:pt-6">
+              <Link href="/">
+                <Image src="/logo.svg" alt="Love Logo" width={75} height={75} />
+              </Link>
+              <Button> Prihlásenie</Button>
+            </div>
+          </div>
+          <div className="grid grid-cols-2 gap-8 sm:gap-6 sm:grid-cols-3 text-center md:text-start">
+            {footerMenuList.map((footerSection, key) => (
+              <div key={key}>
+                <h2 className="mb-6 text-sm font-semibold text-gray-900 uppercase dark:text-white">
+                  {footerSection.header}
+                </h2>
+                {footerSection.items.map((item, itemKey) => (
+                  <ul
+                    key={itemKey}
+                    className="text-gray-500 dark:text-gray-400 font-medium">
+                    <li className="mb-4">
+                      <Link href={item.link} className="hover:underline">
+                        {item.text}
+                      </Link>
+                    </li>
+                  </ul>
+                ))}
+              </div>
+            ))}
+          </div>
+        </div>
+        <hr className="my-6 border-gray-200 sm:mx-auto dark:border-gray-700 lg:my-8" />
+        <div>
+          <div className="flex mt-4 sm:justify-center sm:mt-0">
+            <a
+              href="https://github.com/RomanVelky/kalkulacka"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="text-gray-500 hover:text-gray-900 dark:hover:text-white ms-5">
+              <svg
```

Comment:
SVGčka inlinovať nechceš, zbytočne to zahlcuje priestor. Normálne si vytvor v public foldri assets folder, tam vytvor icons a doňho hoď XXX.svg. Myslím si že defaultne ti nepôjde importovať priamo ikonky, ale keď si nainštaluješ SVGO package (https://github.com/svg/svgo), nastavíš config tak potom budeš môcť spraviť priamo:
```
import SunIcon from '@/assets/icons/sun.svg'
```
a v kóde použiješ ako React komponent
```
<div>
    <SunIcon />
</div>
```

Reply:
Niesom si uplne 100% istý či vieš potom na tie ikonky naviazať Tailwind classy, takže možno ešte pohladaj iné riešenie aby ti to išlo (google SVG in React with Tailwind). Keby si nič nenašiel daj vedieť a niečo vymyslíme 😄

Reply:
dal som to tak